hibike/hibike_process.py

```diff
@@ -119,12 +119,14 @@ def device_read_thread(index, ser, instructionQueue, errorQueue, stateQueue):
                     params_and_values = hm.parse_device_data(packet, hm.uid_to_device_id(uid))
                     stateQueue.put(("device_values", [uid, params_and_values]))
                 else:
-                    print("[HIBIKE] Port %s received data before enumerating!!!" % ser.port)
+                    print("[HIBIKE] Device %s received data before enumerating!!!" % dev)
                     print("Telling it to shut up")
-                    hm.send(ser, hm.make_subscription_request(1, [], 0))
+                    hm.hid_send(dev, hm.make_subscription_request(1, [], 0))
             elif message_type == hm.messageTypes["HeartBeatRequest"]:
                 if uid is not None:
                     instructionQueue.put(("heartResp", [uid]))
+            parse_time += time.time() - before_time
```

Comment:
Additional comment: we've determined that print statements can slow robots down.  Please, remove this code.